primitives/chain-pangolin/src/lib.rs

```diff
@@ -62,6 +62,9 @@ pub const BEST_FINALIZED_PANGOLIN_HEADER_METHOD: &str = "PangolinFinalityApi_bes
 /// Name of the `ToPangolinOutboundLaneApi::message_details` runtime method.
 pub const TO_PANGOLIN_MESSAGE_DETAILS_METHOD: &str = "ToPangolinOutboundLaneApi_message_details";
 
+/// Name of the With-PangolinParachain message pallet instance that is deployed at bridged chains.
+pub const WITH_PANGOLIN_PARACHAIN_MESSAGES_PALLET_NAME: &str = "BridgePangolinParachainMessages";
```

Comment:
This is not belong to Pangolin.

Reply:
https://github.com/darwinia-network/darwinia-bridges-substrate/blob/main/primitives/chain-pangolin-parachain/src/lib.rs#L55

Reply:
In this branch: https://github.com/darwinia-network/darwinia-common/blob/c6c88b1759d345614301882156e5568d83d853cc/node/runtime/pangolin/src/lib.rs#L265

Reply:
You should modify `primitives/chain-pangolin-parachain`.